homeassistant/components/todoist/calendar.py

```diff
@@ -106,6 +108,23 @@
 SCAN_INTERVAL = timedelta(minutes=1)
 
 
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Todoist calendar platform config entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    projects = await coordinator.async_get_projects()
+    labels = await coordinator.async_get_labels()
+
+    entities = []
+    for project in projects:
+        project_data: ProjectData = {CONF_NAME: project.name, CONF_ID: project.id}
+        entities.append(TodoistProjectEntity(coordinator, project_data, labels))
+
+    async_add_entities(entities, update_before_add=True)
```

Comment:
do we need to update before add if we're doing a `first_refresh` of the coordinator in `__init__.py`?

Reply:
Removed.